### PR TITLE
initialized index.js

### PR DESCRIPTION
**Pull request by erinlimbo**

### CODE REVIEW

**Review by michaelbzhu — COMMENTED** [review]
noice

File: src/client/index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { connect, play } from './networking';
+import { startRendering, stopRendering } from './render';
+import { startCapturingInput, stopCapturingInput } from './input';
+import { downloadAssets } from './assets';
+import { initState } from './state';
+import { setLeaderboardHidden } from './leaderboard';
+
+import './css/main.css';
+
+const playMenu = document.getElementById('play-menu');
```
Comment (johnrso) [inline src/client/index.js:10]: i think this is wrong....

Comment (erinlimbo) [reply src/client/index.js:10]: but...

File: src/client/index.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import { connect, play } from './networking';
+import { startRendering, stopRendering } from './render';
+import { startCapturingInput, stopCapturingInput } from './input';
+import { downloadAssets } from './assets';
+import { initState } from './state';
+import { setLeaderboardHidden } from './leaderboard';
+
+import './css/main.css';
```
Comment (johnrso) [inline src/client/index.js:8]: nice

**Review by johnrso — APPROVED** [review]
very cool

**Comment by michaelbzhu** [comment]
noice

**Pull request merged**